### ISSUE TITLE
scale-switch contrast distance too small

### ISSUE THREAD

**Issue by tk-network** [telekom/scale#2088]
**Scale Version**
3.0.0-beta.137

**Framework and version**
vue 3.2.45

**Current Behavior**
The ```scale-switch``` was criticized in our accessibility report. According to this, the contrast distance of the button to the background is too small.

![image](https://github.com/telekom/scale/assets/93334942/380a3830-54c6-40e5-9110-6ac3df23d54b)

**Expected Behavior**
The contrast ratio of non-text elements to the background should be 3:1.

**Desktop (please complete the following information):**
 - OS: MAC OS
 - Browser chrome
 - Version 112.0.5615.137

**Comment by felix-ico** [comment]
@JoschaIco (tagging doesn't seem to work) - do you have any input on this one?

**Comment by JoschaIco** [comment]
Hi @tk-network, thanks for your feedback. We discussed this contrast with the accessibility team, while we designed the switch. They approved the design even though the background contrast is less than 1:3 because of two things we did: 

1. the shadow of the thumb provides enough contrast to be clearly visible
2. we have an extra marker 1/0 that shows the status of the switch

The perception of the element itself might be different if users have color vision deficiency, but it still should be perceivable and understandable.

**Comment by tk-network** [comment]
Hello @JoschaIco,

thank you for your answer.
I will pass this on to our assessors and discuss it with them.
Thank you very much!
